app/src/main/java/cz/quanti/razym/rocketapp/ui/PullToRefresh.kt

```diff
@@ -0,0 +1,139 @@
+@file:OptIn(ExperimentalMaterialApi::class)
+
+package cz.quanti.razym.rocketapp.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.pullrefresh.PullRefreshIndicator
+import androidx.compose.material.pullrefresh.pullRefresh
+import androidx.compose.material.pullrefresh.rememberPullRefreshState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import cz.quanti.razym.rocketapp.presentation.ShowMessageInSnackBar
+import cz.quanti.razym.rocketapp.presentation.UiScreenState
+import cz.quanti.razym.rocketapp.presentation.UiText
+import cz.quanti.razym.rocketapp.ui.theme.RocketappTheme
+
+@Composable
+fun PullToRefresh(
```

Comment:
pull to refresh i UiScreenState bych taky dal do ui modulu

Reply:
Toto je hotove v dalsich vetvich, tak bych to ted uz nepresouval abych si nepridelaval konflikty